src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListForm/ArtworkListForm.tsx

```diff
@@ -14,6 +14,7 @@ export const MAX_NAME_LENGTH = 40
 export const validationSchema = Yup.object().shape({
   name: Yup.string()
     .required(i18n.t("collectorSaves.artworkListForm.fields.name.required"))
+    .matches(/\S+/, { message: "name cannot be empty" })
```

Comment:
```suggestion
    .matches(/\S+/, { message: "Name cannot be empty" })
```

Reply:
Good call, and it reminded me to move this hard-coded text into the translations file instead: https://github.com/artsy/force/pull/12435/commits/1b01618ceea2820c8b5a8a85692de7059f3c5468